frontend/src/views/MainView.vue

```diff
@@ -61,16 +65,29 @@ export default {
       userData: null,
       setupModalVisible: false,
       userInfoModalVisible: false,
-      // TODO grab user data from cookie
+      failedAuthentication: false,
+    }
+  },
+  async mounted() {
+    let token = getAuthToken()
+    if (token) {
+      await postRequest('account/api/validate_auth_token', token).then(
+        (response) => {
+          if (response.user) {
+            this.userSignedIn({ data: response.user, token: response.token })
```

Comment:
This line and [this line](https://github.com/Speuce/JAMCo/blob/40d1ce868593c2552ab925b3befe4020b9a51044/frontend/src/components/signin/GoogleSignin.vue#L32) both call/emit userSignedIn, so I believe it could call twice upon a pre-Cookie sign in. Is this a potential issue?

Reply:
The `GoogleSignin` modal is only rendered if the cookie either doesnt exist or fails to authenticate (since I added the `failedAuthentication` flag). So `userSignedIn` should only be called once - I don't think there's a case where it can be called twice(?)